Samples/ios/SampleAppiOS-Swift/SampleAppiOS-Swift.xcodeproj/project.pbxproj

```diff
@@ -25,6 +25,27 @@
 /* End PBXBuildFile section */
 
 /* Begin PBXContainerItemProxy section */
+		289E15662948EB5A006104D9 /* PBXContainerItemProxy */ = {
```

Comment:
These changes were done to allow the SampleAppiOS-Swift to run. The only real change that was done was setting the iOS to 14 as the SDK requires it. Once that setting is set in XCode, it does these additional changes that can be seen here.